### DISCUSSION TITLE
Unable to open EPUB File

### DISCUSSION THREAD

**Discussion by BDS19591** [sumatrapdfreader/sumatrapdf#3108] [General]
I am unable to open a new EPUB File in Sumatra.  I can open several others with no problem.  I get a message: "Error loading [path\filename]"  This particular file was downloaded from an ACSM file.  It does open with Adobe Digital Editions, so I AM able to read it, however, again, not with Sumatra.  (NOT a fan of Adobe!  And "LOVE" Sumatra Reader!)  Any help would be appreciated!  Thank you!

**Comment by GitHubRulesOK** [comment]
ACSM files often result in a digital rights managed file download that needs ADE decoding/decrypting to read,
 so unless its adapted by other software its is probably not readable by SumatraPDF
